torch/ao/quantization/_learnable_fake_quantize.py

```diff
@@ -75,7 +80,9 @@ def enable_param_learning(self):
 
     @torch.jit.export
     def enable_static_estimate(self):
-        r"""Enables static observer estimates and disables learning of
+        r"""Enable static estimates of quantization parameters.
```

Comment:
Not backward slashes, so raw string decorator here is superflous
```suggestion
        """Enable static estimates of quantization parameters.
```